File: lesson_1/code/task_2_1_malthusian_explicit_discrete.m
% Clean up
    clear, clc

% Population parameters
    P0 = 2.5;   % Intial popoulation 
    r = 0.027;  % Growt rate 2.7%

% Years of interest
    t_task1 = [1, 5, 7.5, 25.78];

% Results
    disp("Years of interest:")
    disp(t_task1)
    disp("Caclulated population size:")
    disp(P0*exp(r*t_task1))

File: lesson_1/code/task_2_1_malthusian_explicit_discrete.py
import numpy as np

P0 = 2.5   # Initial population
r = 0.027  # Growth rate 2.7%

t_task1 = np.array([1, 5, 7.5, 25.78])

print("Years of interest:")
print(t_task1)
print("Calculated population size:")
print(P0 * np.exp(r * t_task1))